README.md

```diff
@@ -1,63 +1,92 @@
- - [Data download and text extraction](docs/user-guide/Download.rst)
-   - Default implementations of download and extraction of Common Crawl, Wikipedia, and ArXiv data
-   - Users can easily customize the download and extraction and extend to other datasets
- - [Language identification and separation](docs/user-guide/LanguageIdentificationUnicodeFormatting.rst)
-   - Language identification with [fastText](https://fasttext.cc/docs/en/language-identification.html) and [pycld2](https://pypi.org/project/pycld2/)
- - [Text reformatting and cleaning](docs/user-guide/LanguageIdentificationUnicodeFormatting.rst)
-   - Fix unicode decoding errors via [ftfy](https://ftfy.readthedocs.io/en/latest/)
- - [Quality filtering](docs/user-guide/QualityFiltering.rst)
-   - Multilingual heuristic-based filtering
-   - Classifier-based filtering via [fastText](https://fasttext.cc/)
- - [Document-level deduplication](docs/user-guide/GpuDeduplication.rst)
-   - Both exact and fuzzy deduplication are accelerated using cuDF and Dask.
-   - For fuzzy deduplication, our implementation follows the method described in [Microsoft Turing NLG 530B](https://arxiv.org/abs/2201.11990).
-  - [Multilingual downstream-task decontamination](docs/user-guide/TaskDecontamination.rst)
-    -  Our implementation follows the approach of [OpenAI GPT3](https://arxiv.org/pdf/2005.14165.pdf) and [Microsoft Turing NLG 530B](https://arxiv.org/abs/2201.11990)
-  - [Distributed data classification](docs/user-guide/DistributedDataClassification.rst)
-    - Multi-node multi-GPU classifier inference
-    - Allows for sophisticated domain and quality classification
-    - Flexible interface for extending to your own classifier network
-  - [Personal identifiable information (PII) redaction](docs/user-guide/PersonalIdentifiableInformationIdentificationAndRemoval.rst)
-    - Idenficiation tools for removing addresses, credit card numbers, social security numbers and more.
-
-These modules are designed to be flexible and allow for reordering with few exceptions. The [NeMo Framework Launcher](https://github.com/NVIDIA/NeMo-Megatron-Launcher) includes prebuilt pipelines for you to start with and modify as needed.
-
-## Learn More
+NeMo Curator is a Python library composed of several scalable data-mining modules, specifically designed for curating Natural Language Processing (NLP) data to train Large Language Models (LLMs). It enables NLP researchers to extract high-quality text from vast, uncurated web corpora efficiently, supporting the development of more accurate and powerful language models.
+
+NeMo Curator leverages [Dask](https://www.dask.org/) and [RAPIDS](https://developer.nvidia.com/rapids) to scale data curation and provide GPU acceleration. The Python library offers easy-to-use methods for expanding the functionality of your curation pipeline while eliminating scalability concerns.
+
+## Key Features
```

Comment:
This reads a bit long to me. Maybe we could retain the original main bullet points/links, but remove the sub-bullet points? What do you think?